packages/vue/src/components/FInteractiveTable/examples/FInteractiveTableStyling.vue

```diff
@@ -0,0 +1,116 @@
+import { LiveExample } from "@forsakringskassan/docs-live-example";
+
+export default defineComponent({
+    name: "FInteractiveTableStyling",
+    components: { LiveExample, FCheckboxField },
+    data() {
+        return {
+            isStriped: false,
+            hasRowDescription: false,
+            hasCustomEmptyText: false,
+            hasHiddenCaption: false,
+            hasRowHeader: false,
+            hasHover: false,
+        };
+    },
+    computed: {
+        livedata(): object {
+            return {
+                items: [
+                    {
+                        id: "1",
+                        level: "Föräldrapenning",
+                        start: "2022-04-11",
+                        end: "2022-04-20",
+                        antal: "10",
+                    },
+                    {
+                        id: "2",
+                        level: "Tillfällig föräldrapenning",
+                        start: "2022-05-02",
+                        end: "2022-05-04",
+                        antal: "3",
+                    },
+                    {
+                        id: "3",
+                        level: "Föräldrapenning",
+                        start: "2022-05-16",
+                        end: "2022-05-27",
+                        antal: "11",
+                    },
+                ],
+            };
+        },
+        components(): object {
+            return { FIcon, FInteractiveTable, FTableColumn };
+        },
+        striped(): string {
+            return this.isStriped ? "striped" : "";
+        },
+        rowHeader(): string {
+            return this.hasRowHeader ? `:row-header="true"` : "";
+        },
+        rowDescription(): string {
+            return this.hasRowDescription ? `description="(åååå-mm-dd)"` : "";
+        },
+        hover(): string {
+            return this.hasHover ? "hover" : "";
+        },
+        caption(): string {
+            return this.hasHiddenCaption
+                ? `<span class="sr-only">Utbetalningar</span>`
+                : "Utbetalningar";
+        },
+        template(): string {
+            return /* HTML */ `
+                <f-interactive-table
+                    :rows="items"
+                    ${this.striped}
+                    ${this.hover}
+                    :showActive="false"
+                    key-attribute="id"
+                >
+                    <template #caption> ${this.caption} </template>
+                    <template #default="{ row }">
+                        <f-table-column name="level" title="Nivå" ${this.rowHeader} type="text">
+                            {{ row.level }}
+                        </f-table-column>
+                        <f-table-column
+                            name="start"
+                            title="Från och med"
+                            ${this.rowDescription}
+                            type="text"
```

Comment:
```suggestion
                            type="date"
```

right?

Eventuellt släng på `v-format:date` också?